x-pack/plugins/security_solution/common/detection_engine/schemas/common/rule_monitoring.ts

```diff
@@ -38,7 +62,7 @@ export const ruleExecutionStatusOrderByStatus: Record<
 > = {
   [RuleExecutionStatus.succeeded]: 0,
   [RuleExecutionStatus['going to run']]: 10,
-  [RuleExecutionStatus.warning]: 20,
+  [RuleExecutionStatus.running]: 15,
```

Comment:
Should we leave these legacy events as comments w/ a deprecation date so we don't accidentally use the same `sortOrder` at some point in the future when there could still be legacy events floating around in a system?

Reply:
Eh, `warning` has been deprecated since pre 8.x and `going to run` is still captured, so nevermind here 👍